.prettierrc.json

```diff
@@ -1,5 +1,5 @@
 {
-  "semi": false,
+  "printWidth": 80,
```

Comment:
it seems [default is 80](https://prettier.io/docs/en/options.html). was there any particular reason for setting it explicitly?

Reply:
@jakoblind 
To setup airbnb eslint standard rules, I used a below command, which interactively asks us to provide printWidth value. 
`exec 3<&1;bash <&3 <(curl https://raw.githubusercontent.com/paulolramos/eslint-prettier-airbnb-react/master/eslint-prettier-config.sh 2> /dev/null)
`
I answered the default and it wrote the same to `.prettierrrc.json`. I think we can remove it as it is default but, it's okay to have it, in case if we want to change it later. If you agree, you can merge this PR.

Reply:
Ok, I see. Let's keep it then. merging